packages/terra-abstract-modal/tests/wdio/abstract-modal-spec.js

```diff
@@ -71,33 +71,31 @@ Terra.describeViewports('Abstract Modal', ['medium'], () => {
     });
   });
 
-  describe('Disabled Close On Esc', () => {
-    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-disable-close-on-esc'));
-
-    Terra.it.validatesElement('open', { selector });
-
-    it('attempts to close modal with ESC', () => {
-      browser.keys('Escape');
-    });
+  it('displays a modal with close disabled on Esc', () => {
+    browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-disable-close-on-esc');
+    Terra.validates.element('disabled close on esc', { selector });
+  });
 
-    it('validates close on escape is disabled and the modal was not removed', () => {
-      expect(browser.isExisting('div[role="dialog"]')).to.be.true;
-    });
+  it('attempts to close modal with ESC', () => {
+    browser.keys('Escape');
   });
 
-  describe('Disabled Close On Outside Click', () => {
```

Comment:
Why remove this describe? All four `it` blocks are testing `'Disabled Close On Outside Click'`

Reply:
Done b9d076f